source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer.xcodeproj/project.pbxproj

```diff
@@ -29,6 +29,9 @@
 		6B096D5122551F35006CC034 /* RichTextBlock.TextRun.SelectActions.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D5022551F35006CC034 /* RichTextBlock.TextRun.SelectActions.json */; };
 		6B096D53225585DD006CC034 /* RichTextBlock.Highlights.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D52225585DD006CC034 /* RichTextBlock.Highlights.json */; };
 		6B096D582257DA0C006CC034 /* Action.IconUrl.TwoActions.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D572257DA0B006CC034 /* Action.IconUrl.TwoActions.json */; };
+		6B096D74225FC51F006CC034 /* SimpleFallback.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D73225FC51F006CC034 /* SimpleFallback.json */; };
+		6B096D76225FF465006CC034 /* DeepFallback.json in Resources */ = {isa = PBXBuildFile; fileRef = 6B096D75225FF465006CC034 /* DeepFallback.json */; };
```

Comment:
Looks like this file has a conflict at this point

Reply:
they have same pre/sufix, but are different numbers.

---
In reply to: [275512370](https://github.com/Microsoft/AdaptiveCards/pull/2675#discussion_r275512370) [](ancestors = 275512370)